lib/tropic/tropic2form.flow

```diff
@@ -28,6 +28,7 @@ makeContext(availableWH : Transform<WidthHeight>) -> TParentInfo {
 	TParentInfo(
 		fwidth(availableWH),
 		fheight(availableWH),
+		getDefaultRtl(),
```

Comment:
is it OK with new feature called TextDirection on the paragraph's level?

Reply:
I will segregate this.